controller/script/simulate-proxy/main.go

```diff
@@ -22,7 +22,99 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
-/* A simple script for posting simulated telemetry data to the proxy api */
+/* A simple script for exposing simulated prometheus metrics */
+
+type simulatedProxy struct {
+	sleep time.Duration
+}
+
+type proxyVectors struct {
+	requestTotals      *prom.CounterVec
+	responseTotals     *prom.CounterVec
+	responseDurationMs *prom.HistogramVec
+	responseLatencyMs  *prom.HistogramVec
+}
+
+// generateProxyTraffic randomly creates metrics under the guise of a single conduit proxy routing traffic.
+func (s *simulatedProxy) generateProxyTraffic(namespace string, podList []*v1.Pod, proxyVec *proxyVectors) {
+	inboundPod := podList[rand.Intn(len(podList))]
+
+	ip := randomPodIp(podList, nil)
+
+	for {
+		inboundRandomCount := int(rand.Float64() * 10)
+		outboundRandomCount := int(rand.Float64() * 10)
+		randomOutboundPod := podList[rand.Intn(len(podList))]
+
+		proxyVec.
+			requestTotals.
+			With(overrideDefaultLabels(prom.Labels{
+				"direction":  "inbound",
```

Comment:
randomize `inbound `/`outbound` (with the caveat that we only (and always) set the `dst_*` labels on `outbound`).